src/shared/lib/MutationUtils.ts

```diff
@@ -365,3 +365,10 @@ export function isValidGenomicLocation(mutation: Partial<Mutation>): boolean {
 
     return false;
 }
+
+export function hasASCNProperty(mutation: any, property: string) {
+    return (
+        mutation.alleleSpecificCopyNumber !== undefined &&
+        (mutation.alleleSpecificCopyNumber as any)[property] !== undefined
```

Comment:
is there a reason this is cast to any?

Reply:
@alisman so the `alleleSpecificCopyNumber` is another type, as opposed to a member of the `mutation` so to code here is checking whether a certain property in the `AlleleSpecificCopyNumber` object is undefined. Attempting to cast it as a `AlleleSpecificCopyNumber`  makes the compiler unhappy and reports this error 

```
Element implicitly has an 'any' type because expression of type 'string' can't be used to index type 'AlleleSpecificCopyNumber'.
```

Do you have any suggestions for working around this or would casting as `any` be okay in this situation?